firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -511,14 +511,6 @@ expected<percent_t> EtbController::getClosedLoop(percent_t target, percent_t obs
 		m_shouldResetPid = false;
 	}
 
-	// Only report the 0th throttle
-	if (m_function == ETB_Throttle1) {
-#if EFI_TUNER_STUDIO
-		// Error is positive if the throttle needs to open further
-		engine->outputChannels.etb1Error = target - observation;
```

Comment:
Help me understand, what's the goal here?  Is this property dead now, then?

Reply:
@nmschulte manually copying values into "outputChannels" is part of the legacy pattern

these days we automatically log some "models" thus on the image you would see same value logged twice

this migration is still not 100% complete and there are probably corner cases. in this specific case it looks like the legacy entry is no longer needed

Reply:
@nmschulte vaguely related to #3614